dashboard/app/models/script.rb

```diff
@@ -333,7 +333,7 @@ def beta?
   end
 
   def self.beta?(name)
-    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5'
+    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5' || name.start_with?('csd')
```

Comment:
My thinking was that this allows curriculum writers to make new csd scripts without worrying about them not being marked as beta. There won't be any CSD scripts that aren't beta until we flip the switch on the whole course for the 17/18 school year.